### ISSUE TITLE
Restangular?  $.param?  Underscore?

### ISSUE THREAD

**Issue by bramski** [alexk111/ph-challenge#4]
So you've chosen to use ngResource, build url parameters by hand, and do a lot of manual looping of arrays.  Have you heard of Restangular?  UnderscoreJS?  They may have saved you a fair bit of work.

**Comment by alexk111** [comment]
Yeah, you're right. My bad. RestAngular would be a great fit for the app. Used ngResource in the challenge app just because when I first saw RestAngular it didn't fit an app I was working on (due to a non-standard "REST API" on the backend or something else).
About arrays looping. The only reason I stick to manual looping is: http://jsperf.com/each-on-array-jquery-vs-underscore-vs-js
